ehrql/measures/measures.py

```diff
@@ -165,19 +164,17 @@ def define_measure(
         # Ensure all required arguments are provided (either directly or by defaults)
         for key, value in kwargs.items():
             if value is None:
-                raise ValidationError(
-                    f"No value supplied for '{key}' and no default defined"
-                )
+                raise Error(f"No value supplied for '{key}' and no default defined")
```

Comment:
What's the reason for not raising a ValidationError? Is it just for user-friendliness?

Reply:
Yeah, it was just a case of not needlessly creating new exception classes for user-facing errors.